src/datachain/lib/dc.py

```diff
@@ -731,6 +745,44 @@ def datasets(
             **{object_name: datasets},  # type: ignore[arg-type]
         )
 
+    @classmethod
+    def studio_datasets(
+        cls,
+        team: Optional[str] = None,
+    ) -> Iterator[DatasetListRecord]:
+        """Get datasets from Studio.
+
+        Parameters:
+            team: Team name.
+
+        Returns:
+            Iterator[DatasetInfo]: Iterator of DatasetInfo objects.
+        """
+        from datachain.remote.studio import StudioClient
+
+        client = StudioClient(team=team)
+
+        response = client.ls_datasets()
+
+        if not response.ok:
+            raise DataChainError(response.message)
+
+        if not response.data:
+            return
+
+        for d in response.data:
+            dataset = DatasetListRecord(
```

Comment:
Is there a better option to form dataset list record from the response dictionary? cc. @dreadatour

Reply:
Thanks, created a from_dict, moved the logic to catalog to return the proper types as well.